validator_test.go

```diff
@@ -0,0 +1,266 @@
+/**
+ *  Copyright (c) 2022  Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package wrp
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func testTypeValidatorValidate(t *testing.T) {
+	type Test struct {
+		m                 map[MessageType]Validators
+		defaultValidators Validators
+		msg               Message
+	}
+
+	var alwaysValid ValidatorFunc = func(msg Message) error { return nil }
```

Comment:
I think we should define and export this in our regular code.  I think it's a reasonable base function consumers might use, and then our example and tests don't have to redefine it too.